src/main/resources/theme/style.css

```diff
@@ -873,13 +873,6 @@
 
 /***************** Login *****************/
 
-.login-root {
```

Comment:
Add back in

Reply:
https://material.io/design/

https://www.google.fr/search?ei=Z9sCXcWLM4GWaJDJn5AJ&q=material+design+background+image&oq=material+design+background+image&gs_l=psy-ab.3..0i19l2j0i22i30i19.133099.135599..135705...0.0..1.131.2751.26j6......0....1..gws-wiz.......0i71j0i131j0j0i67j0i131i67j0i3j0i22i30j0i13i5i30i19.ZzHelNZT_94